packages/benchmark/src/testScenario/testDataLoader.ts

```diff
@@ -0,0 +1,31 @@
+import fs from 'fs';
+import path from 'path';
+import { TestScenario } from '@/types/testScenario';
+import { Workflow } from '@/n8nApiClient/n8nApiClient.types';
+
+/**
+ * Loads test data files from FS
+ */
+export class TestDataFileLoader {
+	async loadTestDataForScenario(testScenario: TestScenario): Promise<{
+		workflows: Workflow[];
+	}> {
+		const workflows = await Promise.all(
+			testScenario.testData.workflowFiles?.map((workflowFilePath) =>
+				this.loadSingleWorkflowFromFile(path.join(testScenario.testScenarioPath, workflowFilePath)),
+			) ?? [],
+		);
+
+		return {
+			workflows,
+		};
+	}
+
+	private loadSingleWorkflowFromFile(workflowFilePath: string): Workflow {
+		const fileContent = fs.readFileSync(workflowFilePath, 'utf8');
+
+		const workflow = JSON.parse(fileContent);
+
+		return workflow;
+	}
```

Comment:
A typo in the JSON might lead to an unclear error. Elsewhere we're using `jsonParse` from `n8n-workflow`. There's also a lint rule enforcing this, but linting setup in this package seems to be skipping it.